pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -536,4 +539,28 @@ func init() {
 	if err != nil {
 		panic(err)
 	}
+
+	err = InfraSAs.addServiceAccount(
+		InfraGCControllerServiceAccountName,
+		authorizationapi.ClusterRole{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: GCControllerRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				// GCController.podStore.ListWatch
+				{
+					Verbs:     sets.NewString("list", "watch"),
```

Comment:
Specify `APIGroups: sets.NewString(kapi.GroupName)` here and on the other rule.

@liggitt I'd rather show permissions per resource, I think its easier to read.  Do you object?  The location falls out of date quickly.